### ISSUE TITLE
[terra-avatar] Enhancement for multi-user avatars

### ISSUE THREAD

**Issue by naveenajain** [cerner/terra-core#3294]
# Feature Request

## Description
Currently, Terra Avatar does not provide a way to display images/initials for multiple users at once. Having conversation with multiple people in a group chat should have a way to display either images or initials of the users who are included in the chat.

We would like to have an enhancement for multi-user avatars (both with images and with user initials) so that they would have an indication who the users are.

**Comment by bb3284** [comment]
I thought I remember seeing a design in the past where "skittles" would be used for such multi-user avatars.  What ever happened with that?

**Comment by naveenajain** [comment]
I didn't find any skittles in [Terra Avatar Component](https://engineering.cerner.com/terra-ui/components/terra-avatar/avatar/avatar). 

Can someone from Terra team can look into it.

**Comment by naveenajain** [comment]
Tagging @ryanthemanuel for visibility.

**Comment by ryanthemanuel** [comment]
@naveenajain and @bb3284 we had a discussion about this between UX foundations and terra's engineering leadership in our planning meeting today and unfortunately, this is not something that is going to be picked up as an addition to terra. The number of use cases for this addition is not broad enough to be incorporated at this point. The recommendation is to work with your UX designer to figure out potential solutions with avatar as it is or to create a custom component that meets your needs.

**Comment by bb3284** [comment]
I would generally disagree that this doesn't have broad enough need to be useful.  We are planning to create a Messaging component in the near future that can be consumed across other orion applications such as video visits and potentially in even Healthe Intent applications.  The need for a meaningful multi-use avatar is basic functionality for group conversations in such use cases. 

Also note we have a problem today where since Care Team component uses the terra avatar and is consumed in messenger, even in our own application we have variance on the avatars (colors are different for the same user as viewed in care team vs messenger), so we have need to move to this component, however the lack of a multi-use avatar prevents doing so.
